lively.morphic/components/policy.js

```diff
@@ -205,18 +297,34 @@ export class StylePolicy {
     if (policyOrDescriptor.isPolicy || policyOrDescriptor.isComponentDescriptor) {
       this._parent = policyOrDescriptor;
     } else {
-      const { click, hover, light, dark, breakpoints, auto = this._parent } = policyOrDescriptor;
-      this._parent = auto;
-      this._autoMaster = auto;
+      const {
+        click, hover, light, dark, breakpoints, states,
+        auto = this._parent, isSplitInline = false
+      } = policyOrDescriptor;
+      this._isSplitInline = isSplitInline;
```

Comment:
This is a little complicated, but I will try: Before policy states we basically just ad `click` `hover` and `auto` to switch between different masters that are applied to a morph via the policy.
The most tricky thing about policies are inline policies (remember those?). Inline Policies are basically policies *within* policies. You get this by using `part()` within the submorphs of one of the morphs inside the `component(...)`. 
These inline policies can quickly become complex, so I try to keep it simple here: Previously we would just leave inline policies where they are and just toggle the master of the policy that is affected by the event state. (imagine things like clicked button, or hover effects). This worked "well" since events are usually confined to a small area in the scene where nested inline policies just most of the time did not even appear.
With component states however we can no longer afford to leave the inline policies untouched: Imagine toggling between dark and light mode. We cant just leave the inline policies where they are, since they will have completely incorrect hover or click effect if they are just left as is! Again, if you *theoretically* would have attempted to toggle between dark and light mode based on hover, you would have run into similar issues, but no one does that, so it did not come up as an issue.
With that in mind, I needed a way to essentially make inline policies aware of the fact that they are "split" into a version that applied during dark and one that applies during light mode (inject your favorite state partitioning here).
A split inline policy will essentally "swap" itself based on the state of the root style policy.
This is what is meant by being "split inline". We can discuss a better name, but that is essentially the concept.

Reply:
rename `isSplitInline`  to `statePartitionedInline`